kitsune/search/v2/es7_utils.py

```diff
@@ -19,7 +21,11 @@ def _insert_custom_filters(locale, filter_list, char=False):
 
     def mapping_func(filter):
         if type(filter) is dict:
-            name = f'{locale}_{filter["type"]}'
+            # sort filter dict by keys, so reordering doesn't change hash
+            sorted_filter = {key: filter[key] for key in sorted(filter)}
```

Comment:
wouldn't be simpler to tune the naming schema instead?

Reply:
The only common key across all filters is `"type"`. Although, I guess naming them based on their position in the locale's filter list would work. I'll try that.